magstock/site_sections/magstock.py

```diff
@@ -24,7 +24,20 @@ def grouped(self, session):
                 except:
                     pass
 
-        return {'grouped': sorted({frozenset(group) for group in lookup.values()}, key=len, reverse=True)}
+        def match(a):
+            return ((not site or a.site_type == int(site))
```

Comment:
Hey, isn't there an extra ( here? There's 3 to starting parentheses only 2 ending parentheses.